config/clients/python/CHANGELOG.md.mustache

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.2.1
+
+### [0.2.1](https://{{gitHost}}/{{gitUserId}}/{{gitRepoId}}/compare/v0.2.0...v0.2.1) (2023-09-05)
+- fix(client): fix a crash when calling check with contextual tuples
+- chore: fixed a few issues in docs
```

Comment:
fixed a few issues in docs seems a bit weird - perhaps just mention doc cleanup or something like that?